modules/nodes-pods-priority-preempt-about.adoc

```diff
@@ -17,6 +17,13 @@ Preemption does not necessarily remove all lower-priority pods from a node. The
 
 The scheduler considers a node for pod preemption only if the pending pod can be scheduled on the node.
 
+[id="non-preempting-priority-class_{context}"]
+== Non-preempting priority class ==
```

Comment:
Remove the ending `==`:

```
 == Non-preempting priority class
```

Reply:
@bergerhoffer can we add (Tech-preview) here as well ?

Reply:
@kasturinarra Good suggestion. This PR is closed, but I've made this update in my new PR here: https://github.com/openshift/openshift-docs/pull/27727